src/docs/sphinx/advancedExamples/validationStudies/isothermalLeakyWell/Example.rst

```diff
@@ -0,0 +1,404 @@
+since we follow the approach of
+`(Class et al., 2009) <https://link.springer.com/article/10.1007/s10596-009-9146-x>`__
+and do not simulate flow in the aquitard, as we will see in the next section.
+
+.. note::
+   Since the two aquifer regions share the same material properties, they could have been defined as a single ``CellElementRegion`` for a more concise XML file producing the same results. The same is true for the two well regions.  
+
+------------------------
+Flow solver
+------------------------
+
+The isothermal immiscible simulation is performed by the GEOSX general-purpose multiphase
+flow solver based on a TPFA discretization defined in the XML block **CompositionalMultiphaseFVM**:
+
+.. literalinclude:: ../../../../../../inputFiles/compositionalMultiphaseFlow/benchmarks/isothermalLeakyWell/isothermalLeakyWell_benchmark.xml
+    :language: xml
+    :start-after: <!-- SPHINX_SOLVER -->
+    :end-before: <!-- SPHINX_SOLVER_END -->
+
+We use the ``targetRegions`` attribute to define the regions where the flow solver is applied.
+Here, we follow the approach described in
+`(Class et al., 2009) <https://link.springer.com/article/10.1007/s10596-009-9146-x>`__
+and do not simulate flow in the aquitard, considered as impermeable to flow.
+We only simulate flow in the two aquifers (``aquiferTop`` and ``aquiferBottom``), in the bottom
+part of the injection well (``injectionWell``), and in the leaky well connecting the two aquifers (``leakyWell``).
+
+We list the corresponding solid names and permeability names in the attributes ``solidNames``
+and ``permeabilityNames``, respectively.
+The same fluid and relative permeability models are used in all target regions, which is why the
+attributes ``fluidNames`` and ``relPermNames`` contain only one entry.
+
+------------------------------
+Constitutive laws
+------------------------------
+
+This benchmark test involves an immiscible, incompressible, two-phase model.
+The best approach to represent this fluid behavior in GEOSX is to use the **DeadOilFluid** model,
+although this model was initially developed for simple isothermal oil-water or oil-gas systems
+(note that this is also the approach used for the Eclipse simulator, as documented in
+`(Class et al., 2009) <https://link.springer.com/article/10.1007/s10596-009-9146-x>`__). 
+
+.. literalinclude:: ../../../../../../inputFiles/compositionalMultiphaseFlow/benchmarks/isothermalLeakyWell/isothermalLeakyWell_benchmark.xml
+    :language: xml
+    :start-after: <!-- SPHINX_FLUID -->
+    :end-before: <!-- SPHINX_FLUID_END -->
+
+The fluid properties (constant densities and constant viscosities) are those given in the
+article documenting the benchmark.
+To define an incompressible fluid, we set of the formation volume factors to 1.  
+		 
+.. note::		 
+   For now, we have to use the label ``oil`` to denote the CO2 phase in the Dead-Oil model, but this will be changed in future GEOSX releases.
```

Comment:
Did you create an issue for this?  Otherwise we will probably forget.

Reply:
Yes, I will create an issue with the problems I faced working on the benchmarks